templates/service-rabbitmq.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```

Comment:
Same here as well.  We should use the openstack-helm rabbitmq chart and improve that with resiliency.